sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/GetUserByUserIdTest.java

```diff
@@ -21,10 +21,7 @@
   public Object[][] getUserByUserIdFailure() {
     return new Object[][] {
       new Object[] {
-        "testGetUserByUserIdFailureWithoutAuthToken",
-        false,
-        "invalidUserId",
-        HttpStatus.UNAUTHORIZED
+        "testGetUserByUserIdFailureWithoutAuthToken", false, "invalidUserId", HttpStatus.NOT_FOUND
```

Comment:
Why is it not found when auth token is not provided? Shouldn't it be unauthorized?